tests/unit/test_run_docker.py

```diff
@@ -18,7 +18,7 @@ def test_query_loop_correct_interval(
         mock_async_dynatrace_gcp_extension: AsyncMock,
         mock_sfm_send_loop_timeouts: AsyncMock,
     ):
-    mock_metrics_pre_launch_check.return_value = run_docker.PreLaunchCheckResult(projects=[], services=[])
+    mock_metrics_pre_launch_check.return_value = run_docker.PreLaunchCheckResult(services=[])
```

Comment:
Why were projects removed from the PreLaunchCheckResult? I'm struggling to grasp this change.